src/abstractdataframe/io.jl

```diff
@@ -202,6 +202,60 @@ end
 
 ##############################################################################
 #
+# LaTeX output
+#
+##############################################################################
+
+function latex_char_escape(char::AbstractString)
+    if char == "\\"
+        return "\\textbackslash{}"
+    elseif char == "~"
+        return "\\textasciitilde{}"
+    else
+        return string("\\", char)
+    end
+end
+
+function latex_escape(cell::AbstractString)
+    cell = replace(cell, ['\\','~','#','$','%','&','_','^','{','}'], latex_char_escape)
+    return cell
+end
+
+function Base.show(io::IO, ::MIME"text/latex", df::AbstractDataFrame)
+    nrows = size(df, 1)
+    ncols = size(df, 2)
+    cnames = _names(df)
+    alignment = repeat("c", ncols)
+    write(io, "\\begin{tabular}{r|")
+    write(io, alignment)
+    write(io, "}\n")
+    write(io, "\t& ")
+    header = join(map(c -> latex_escape(string(c)), cnames), " & ")
+    write(io, header)
+    write(io, "\\\\ \n")
+    write(io, "\t\\hline \n")
+    for row in 1:nrows
+        write(io, "\t")
+        write(io, @sprintf("%d", row))
+        for col in 1:ncols
+            write(io, " & ")
+            cell = df[row,col]
+            if !isnull(cell)
```

Comment:
Sorry, I hadn't noticed this. This should print `#NULL` instead of nothing for null values, just like at the REPL and in HTML. Could you apply the strategy I described at https://github.com/JuliaStats/DataFrames.jl/pull/1084 in a new PR?